test/watchCases/plugins/extract-text-plugin/0/index.js

```diff
@@ -0,0 +1,6 @@
+import css from "./style.css"
+
+it("should ignore change to file and directory", function() {
+	// nothing to assert
+	// this test would hang if stats.hash didn't update upon css change
```

Comment:
I thought about it, but I'm not sure how to assert. You obviously cannot assert specific hash in a file that is included in that hash computation :)
So in this case, we actually need to assert whether stats.hash changed between the runs, so we'd need to pass the whole history of stats to do that. Are you ok with that?